app/src/main/java/com/appunite/loudius/common/Screen.kt

```diff
@@ -69,5 +70,19 @@ sealed class Screen(val route: String) {
             pullRequestNumber: String,
             submissionDate: String,
         ): String = "reviewers_screen/$pullRequestNumber/$owner/$repo/$submissionDate"
+
+        fun getInitialValues(savedStateHandle: SavedStateHandle) = ReviewersInitialValues(
+            checkNotNull(savedStateHandle[ownerArg]),
+            checkNotNull(savedStateHandle[repoArg]),
+            checkNotNull(savedStateHandle[pullRequestNumberArg]),
+            checkNotNull(savedStateHandle[submissionDateArg]),
```

Comment:
suggestion: maybe we could parse submission time to `LocalDateTime` already in this function

Reply:
Sounds good for me - added 👌